src/data/sidebar.ts

```diff
@@ -121,6 +121,23 @@ export const sidebarContent: ISidebarContent = [
       makePage("MongoDB", "databases", ["database", "nosql"]),
     ],
   },
+  {
+    title: "Troubleshoot",
+    pages: [
+      makePage("Fixing Common Errors", "troubleshoot", [
+        "error",
+        "errors",
+        "railway error",
+        "railway errors",
+        "application error",
+        "failed to respond",
+        "application failed to respond",
+        "503",
+        "service unavailable",
+        "503 service unavailable",
+      ]),
```

Comment:
Went a bit too verbose on this. I want to make sure this is reachable via search.

This can go away after #225 lands